draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -6421,8 +6384,8 @@ Expect: 100-continue
 </t>
 <t>
    A <x:dfn>protection space</x:dfn> is defined by the canonical root URI (the
```

Comment:
Note to self: this can now be replaced with origin and a link to the new section.